api/handlers.go

```diff
@@ -39,17 +40,29 @@ func (api *API) userUploadsHandler(w http.ResponseWriter, req *http.Request, ps
 		WriteError(w, err, http.StatusInternalServerError)
 		return
 	}
-	offset, _ := strconv.Atoi(ps.ByName("offset"))
-	limit, _ := strconv.Atoi(ps.ByName("limit"))
-	ups, err := api.staticDB.UploadsByUser(req.Context(), *u, offset, limit)
+	if err = req.ParseForm(); err != nil {
+		WriteError(w, err, http.StatusBadRequest)
+	}
+	offset, err1 := fetchOffset(req.Form)
+	limit, err2 := fetchLimit(req.Form)
+	if err = errors.Compose(err1, err2); err != nil {
+		WriteError(w, err, http.StatusBadRequest)
+	}
+	ups, total, err := api.staticDB.UploadsByUser(req.Context(), *u, offset, limit)
 	if err != nil {
 		WriteError(w, err, http.StatusInternalServerError)
 	}
-	WriteJSON(w, ups)
+	response := database.UploadsResponseDTO{
```

Comment:
I think we need to return the `limit` and the `offset`, the user requested those params so no need to return them.
Also if we call the param `limit`, I wouldn't call the return variable `PageSize`?

Reply:
`limit` renamed to `pageSize` for consistency.

If @kwypchlo doesn't need the `pafeSize` and `offset` in the response, I'll remove them.